src/coreclr/jit/emitriscv64.cpp

```diff
@@ -700,7 +700,7 @@ void emitter::emitIns_R_R_R(
     if ((INS_add <= ins && ins <= INS_and) || (INS_mul <= ins && ins <= INS_remuw) ||
         (INS_addw <= ins && ins <= INS_sraw) || (INS_fadd_s <= ins && ins <= INS_fmax_s) ||
         (INS_fadd_d <= ins && ins <= INS_fmax_d) || (INS_feq_s <= ins && ins <= INS_fle_s) ||
-        (INS_feq_d <= ins && ins <= INS_fle_d))
+        (INS_feq_d <= ins && ins <= INS_fle_d) || (INS_lr_w <= ins && ins <= INS_amomaxu_d))
```

Comment:
Seems lesser atomic instruction is `INS_amoadd_w`, not `INS_lr_w`.

Reply:
`INS_lr_w` comes first. `INS_*` is an enum, not the instruction itself.